chart/templates/aws-cloudwatch-metrics.yaml

```diff
@@ -23,7 +23,9 @@ spec:
     namespace: amazon-cloudwatch
   syncPolicy:
     automated:
-      prune: true    
+      prune: true
+    syncOptions:
+      - CreateNamespace={{ .Values.awsCloudWatchMetrics.createNamespace }}
```

Comment:
this looks like its created here https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/db9c72a6c881e6078ece008fd239fbfc02b202fb/modules/kubernetes-addons/aws-cloudwatch-metrics/locals.tf#L39